src/admin/bundles/bundles-list.html

```diff
@@ -1,5 +1,24 @@
 <motech-list>
-    <motech-list-item ng-repeat="bundle in bundles" >
+    <motech-list-header>
+        <form class="form-inline ">
+            <div class="form-group">
+                <label for="admin.bundles.list.filter">{{msg('admin.filter')}}</label>
```

Comment:
Shouldn't it be `{{ 'admin.filter' | translate}}` here and in all of the below lines?